changelog/17.0/17.0.0/summary.md

```diff
@@ -49,6 +50,16 @@ Here is a (condensed) example of stats output:
 }
 ```
 
+
+#### <a id="keyspace-name-validation"> Keyspace name validation in TopoServer
+
+Prior to v17, it was possible to create a keyspace with invalid characters, which would then be inaccessible to various cluster management operations.
+
+Now, the TopoServer's `GetKeyspace` and `CreateKeyspace` methods return an error if given an invalid name.
```

Comment:
This seems overly broad. We don't know what other invalid characters people can provide that break in unpredictable ways. The only one we are disallowing is `/`.
I would remove the first two sentences altogether and just leave the last one.